src/rp2_common/hardware_dma/include/hardware/dma.h

```diff
@@ -536,6 +536,48 @@ static inline void dma_set_irq1_channel_mask_enabled(uint32_t channel_mask, bool
     }
 }
 
+/*! \brief  Determine if a particular channel is a cause of DMA_IRQ_0
+ *  \ingroup hardware_dma
+ *
+ * \param channel DMA channel
+ * \return true if the channel is a cause of DMA_IRQ_0, false otherwise
+ */
+static inline bool dma_channel_get_irq0_status(uint channel) {
+    check_dma_channel_param(channel);
+    return dma_hw->ints0 & (1u << channel);
+}
+
+/*! \brief  Determine if a particular channel is a cause of DMA_IRQ_1
+ *  \ingroup hardware_dma
+ *
+ * \param channel DMA channel
+ * \return true if the channel is a cause of DMA_IRQ_1, false otherwise
+ */
+static inline bool dma_channel_get_irq1_status(uint channel) {
+    check_dma_channel_param(channel);
+    return dma_hw->ints0 & (1u << channel);
```

Comment:
ints0 -> ints1 ?

Reply:
**bump**
If `dma_channel_get_irq0_status` is checking `dma_hw->ints0` then shouldn't `dma_channel_get_irq1_status` be checking `dma_hw->ints1` ?

Reply:
yeah, i haven't gotten around to fixing stuff here yet, i just rebased the branch

Reply:
This is marked as resolved, but doesn't seem to be fixed?